src/style-spec/reference/v8.json

```diff
@@ -3183,6 +3225,24 @@
           }
         }
       },
+      "pitch": {
+        "doc": "Gets the current pitch in degrees. `[\"pitch\"]` may only be used in the `filter` expression for a `symbol` layer.",
```

Comment:
Nit: consider "Returns" instead of "Gets"

Reply:
This is consistent with the wording for `["zoom"]`, do you think we should change both?

Reply:
Yes, I think "Returns" is slightly more clear.